raiden/network/pathfinding.py

```diff
@@ -121,12 +124,27 @@ def as_json(self) -> Dict[str, Any]:
         return data
 
 
+USER_AGENT_STR = (
```

Comment:
I'd love to get a second pair of eyes on this part. Any thoughts about the formatting, @palango @ulope ?

Reply:
I like it!

Maybe adding `DB:` or some such in front of the db version could be nice...